src/lib/components/Toggle/Toggle.js

```diff
@@ -65,6 +30,46 @@ ToggleComponent.defaultProps = {
   overridableId: '',
 };
 
+const Element = ({ overridableId, ...props }) => {
```

Comment:
I think this is redundant...I guess you did it to use the `useContext` hook. If so, what about just defining the `ToggleComponent` class to a function?

Reply:
Fair point. Not sure what I was thinking to keep the class component :sweat_smile:. Thanks!